web/js/doppelkopf.js

```diff
@@ -5,19 +5,42 @@ function addPlayers(select, players) {
     option.appendChild(document.createTextNode(player.name));
     select.appendChild(option);
   }
+  for (let i = 1; i < 5; i += 1) { // Vorauswahl auf empty setzen
+    document.getElementById(`player${i}`).selectedIndex = -1;
+  }
 }
 
 window.onload = function onLoad() {
   const request = new XMLHttpRequest();
 
   request.open('GET', '/api/players');
   request.addEventListener('load', function onRequestLoad() {
-    if (request.status >= 200 && request.status < 300) {
-      const players = JSON.parse(request.responseText);
-      addPlayers(document.getElementById('player'), players);
-    } else {
-      console.warn(request.statusText, request.responseText);
+    for (let i = 1; i < 5; i += 1) { // 4 Spieler können ausgewählt werden
+      if (request.status >= 200 && request.status < 300) {
```

Comment:
Der request ändert seinen Status nicht zwischen Schleifendurchläufen, also musst du das auch nicht bei jedem Schleifendurchlauf überprüfen. Rein wäre es wahrscheinlich logischer, das `if` und das `for` umzudrehen, also:

```
if (request.status >= 200 && request.status < 300) {
   for (let i = 1; i < 5; i += 1) {
      ...
```